packetizer.go

```diff
@@ -41,6 +43,24 @@ func NewPacketizer(mtu int, pt uint8, ssrc uint32, payloader Payloader, sequence
 	}
 }
 
+func (p *packetizer) EnableAbsSendTime(value int) {
+	p.AbsSendTime = value
```

Comment:
Is this necessary if you can just set it directly on the struct?

Reply:
Not really, you're right.

Reply:
wait-- I added it to the interface. `packetizer` isn't exposed.

Reply:
Hmm wonder why Packetizer is an interface in the first place...

Question for a future pull request.